dist/rules/no-internal.js

```diff
@@ -9,6 +9,9 @@
 
 const { getParserServices } = require("./utils/parser");
 const ts = require("typescript");
+const path = require("path");
+const jsdoc = require("../configs/jsdoc");
```

Comment:
Where is this jsdoc config import being used? i dont see its usage here

Reply:
Yeah this must have been left over from some testing, I'll remove